### ISSUE TITLE
Have delegates be able to see a list of unannounced competitions.

### ISSUE THREAD

**Issue by Jambrose777** [thewca/worldcubeassociation.org#1320]
I would like to point out that delegates already have this ability (and I would like it to stay), but it would be beneficial to be able to see them all in a list. They can do this by creating a competition and setting the time and place to wherever and whenever to see any unannounced competition. So the feature is already there for us to use.

I'm not sure how hard it would be to make (although I'm pretty confident that it is already made, just not view-able), but if it is not made, then I would be willing to look into it.

Thoughts?

**Comment by jfly** [comment]
> setting the time and place to wherever and whenever to see any unannounced competition

Nit: This is only for unannounced competitions that have a time and location. Not all of them do.

> Although I'm pretty confident that it is already made, just not view-able

It's interesting to me that you think this, because it implies you think this would be useful. Such a page does not exist =) Can you explain what you think it would be useful for?

**Comment by Jambrose777** [comment]
I specifically use it to post planned competitions that are not ready to be announced. Delegates in my area are able to see these whenever they create a competition thus they know what I'm planning. They are also starting to do this and it has already avoided conflicts in certain areas. Having a page accessible for all of us would make it to where delegates can plan better on a broader scale. (i.e. Texas, virginia, Georgia, and Florida are all having comps on the April 28-29th weekend; being able to see this further ahead of time could avoid all the comps being on the same day.

**Comment by jfly** [comment]
Got it. I have no data to back this up, but I fear that not all delegates use the system as carefully as you describe... for all we know, delegates create competitions and abandon them, or put them in random locations and times that have no bearing on reality.

That said, if you want to create a page that lists these, I won't oppose that. Perhaps a "include unannounced" checkbox on https://www.worldcubeassociation.org/competitions, rather than building a new page? I would only want this "include unannounced" feature to be accessible to delegates or higher, however.

**Comment by SAuroux** [comment]
I don't like this proposal. Wouldn't it be way simplier to just maintain a
Google Doc with all delegates that are actively willing to exchange this
kind of information? That's how we do it in Germany and it allows to
exchange information as soon as there is only a rough idea of it, i.e.
before entering it to the WCA website is possible.

2017-02-14 3:54 GMT+01:00 Jeremy Fleischman <notifications@github.com>:

**Comment by jfly** [comment]
@SAuroux, I see the appeal of a dedicated website for this: it can do distance calculations for you and check for overlapping events. This is kind of painful to do manually. This would not allow for the communication a google doc provides, so I'd see this as something complementary to a google doc rather than replacing it.

My region (the SF Bay Area) needs a system for this, right now we just route every competition request through Ryan Jew.

**Comment by SAuroux** [comment]
Ok, I think I see the benefit. But I kind of dislike exposing all
unannounced competitions of all delegates to all other delegates. I would
really like to see such a feature somehow limited to the desired use case.

2017-02-14 18:14 GMT+01:00 Jeremy Fleischman <notifications@github.com>:

**Comment by jfly** [comment]
> But I kind of dislike exposing all unannounced competitions of all delegates to all other delegates.

As @Jambrose777 said in the original post, this data is *already* exposed to all delegates.

**Comment by Jambrose777** [comment]
@SAuroux I've tried to implement a google doc in the past with no success, but I'm willing to try again. The Issue is that it would be a lot of delegates and it would only pertain to some of the competitions that a delegate would delegate for. i.e. SE USA competitions wouldn't include most of Michigan comps (James H.) but he could still delegate fro Kentucky. This is where the google doc Idea fell apart and I reverted to using WCA and directing delegates to look there. This feature being available would just make the process easier.

Is there a reason that this information should NOT be "exposed" to delegates? They are trusted people within WCA, so what is there to fear?

**Comment by SAuroux** [comment]
@jfly No, it is clearly not exposed in a way which is thought about here. Delegates can only see unannounced competitions **around** competitions they have added. To see all unannounced competition worldwide, a delegate would need to set countless different locations to gather all this data first, which would still only give a snapshot overview of the current situation.

@Jambrose777 I think I realize that the Google Doc solution only works for smaller areas. Indeed, Germany and the US cannot really be compared in this regard.
Yes, for me there are reasons why this information should NOT be exposed to all delegates. In my view, data about unannounced competitions is sensible information. I understand that having access to US data would be helpful for US delegates, but the state of unannounced competitions in Brazil for example is none of a US delegate's business. 
Yes, delegate are trusted people within the WCA, but only to a certain extent. The WCA trusts delegates to supervize WCA competitions, but still requires a report for each of them as a reinsurance and each delegate has to go through a candidate term first. With the same argument, you could ask why not every delegate has access to the WCA database or to the website admin codes. The baseline is, that it is never a good idea to share sensible information to people that do not necessarily need them.

**Comment by AlbertoPdRF** [comment]
Maybe, after #4222 is handled, a good compromise for this issue would be to only show a list of all unannounced competitions in a Delegate's `subregion_id`?

**Comment by dunkOnIT** [comment]
I think there are legitimate use cases in this thread that the website still doesn't solve - perhaps it is as simple as adding a "Show Unannounced" filter in the Competition List for delegates, and then allowing them to further filter either by distance or by country/region
